media/css/base/menu-resp.less

```diff
@@ -186,11 +185,6 @@ html[dir="rtl"] #nav-main .submenu {
         padding: 0;
     }
 
-    .js #nav-main li .submenu {
-        xleft: 80px;
-        xtop: auto;
```

Comment:
No idea - it was so mysterious I even searched in case there was some random proprietary thing I didn't know existed. Came up blank